src/controllers/DrawingModeController.js

```diff
@@ -287,6 +287,8 @@ class DrawingModeController extends AnnotationModeController {
                 if (eventData && eventData.location) {
                     // $FlowFixMe
                     this.currentThread.handleStart(eventData.location);
+                } else {
```

Comment:
Will only set currentThread to undefined if the drawing wasn't saved as a part of a page change